src/main/resources/db/migration/db.feedback.migration/V202409231113__create_company_and_employee_tables.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE company
+(
+    id BIGSERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL
+);
+
+
+CREATE TABLE employee (
+    id BIGSERIAL PRIMARY KEY,
+    first_name VARCHAR(255) NOT NULL,
+    last_name VARCHAR(255) NOT NULL,
+    role VARCHAR(50) CHECK (role IN ('admin', 'manager', 'employee')) NOT NULL,
+    company_id BIGINT REFERENCES company(id),
```

Comment:
let's remove the "check" here, and do the validation in the code. I know Eli sent you this script, let's modify it.
Aslo, please put company_id as the second field, right after the "id"

Reply:
also, get rid of the `References`

Reply:
fixed, mistakingly though that you said that what Eli did was ok with the reference, hence I didn't change it